src/pages/home/report/ContextMenu/genericReportActionContextMenuPropTypes.js

```diff
@@ -1,12 +1,14 @@
 import PropTypes from 'prop-types';
-import reportActionPropTypes from '../reportActionPropTypes';
 
 const propTypes = {
     /** The ID of the report this report action is attached to. */
     reportID: PropTypes.string.isRequired,
 
-    /** The report action this context menu is attached to. */
-    reportAction: PropTypes.shape(reportActionPropTypes).isRequired,
+    /** The ID of report action this context menu is attached to. */
+    reportActionID: PropTypes.string.isRequired,
+
+    /** The ID of the current report of this report action is attached to. */
```

Comment:
Could you reword this comment, it's a bit unclear how it's different from the one for `reportID`

> /** The ID of the report this report action is attached to. */